memcached-spring-boot-autoconfigure/src/test/java/io/sixhours/memcached/cache/MemcachedCacheManagerTest.java

```diff
@@ -85,4 +90,62 @@ public void whenGetDisabledCacheThenIncreaseCacheSize() {
 
         assertThat(cache).isInstanceOf(NoOpCache.class);
     }
+
+	@Test
+	public void whenGetDisabledCacheThenNoOpCache() {
+        MemcachedCacheProperties.CacheConfig cacheConfig = new MemcachedCacheProperties.CacheConfig();
+        cacheConfig.setExpiration(Duration.ofHours(3));
+        cacheConfig.setMetricsEnabled(true);
+        cacheConfig.setDisabled(true);
+
+        Map<String, MemcachedCacheProperties.CacheConfig> configurationPerCache = Stream.of(new Object[][] {
```

Comment:
Couldn't you use the `Collections.singletonMap()`

Reply:
True, I'll change it